functions/index.js

```diff
@@ -250,6 +252,23 @@ async function snapshotData(country) {
     predefinedAcl: "publicRead",
   });
 
+  // Build a simple csv file
+  const csvFileRef = admin.storage().bucket().file(csv_filename);
+  let csv_file_list = [];
+  for (let i = 0; i < data.values.length; i++) {
+      row_string = data.values[i].join(",");
```

Comment:
You may need to handle quotes, newlines, and commas in the values. "Accepting" can contain commas. "Address" and "Instructions" can contain newlines, quotes, and all sorts of other nonsense.

Reply:
I've switched it--that wasn't my comment.

Also, my job requires lots of parsing CSVs, and I personally detest using a `"` as an escape character. The quote character and the escape character should never be the same!

Anyway, I relent. See new revision.